gnovm/pkg/gnolang/helpers.go

```diff
@@ -26,6 +27,11 @@ func IsRealmPath(pkgPath string) bool {
 		!ReGnoRunPath.MatchString(pkgPath)
 }
 
+// IsPackagePath determines whether the given pkgpath is for a package.
+func IsPackagePath(pkgPath string) bool {
```

Comment:
In my view, the distinction in Gno between "realms" (<domain>/r) and standard packages (<domain>/p) adds an extra layer that can be confusing, as it diverges from the traditional Go concept of a package as a simple collection of related source files.